src/banks/registries/directory.py

```diff
@@ -26,11 +32,12 @@ def __init__(self, directory_path: Path, *, force_reindex: bool = False):
             raise ValueError(msg)
 
         self._path = directory_path
-        self._index_path = self._path / "index.json"
+        self._index_path = self._path / DEFAULT_INDEX_NAME
         if not self._index_path.exists() or force_reindex:
             self._scan()
         else:
             self._load()
+        self._meta_path = self._path / DEFAULT_META_PATH
```

Comment:
Any reason why we want to use a separate index for the metadata? If we extend the `PromptFile` model to include `meta: dict[str, Any]` we would piggy back on the existing get/set infraastructure

Reply:
I agree with this, I would make the changes.